test-end-to-end/webapp-cypress/cypress/e2e/003_005_admin_or_teacher_and_content_via_file.cy.js

```diff
@@ -0,0 +1,81 @@
+describe('003_005_admin_or_teacher_and_content_via_file', () => {
+it('Start Ustad Test Server ', () => {
+  // Start Test Server
+  cy.ustadStartTestServer()
+})
+
+it('Admin able to add content block from library', () => {
+ // Admin user login
+  cy.ustadClearDbAndLogin('admin','testpass')
+  cy.ustadAddContentToLibrary('../test-files/content/H5p_Content.h5p','Content_001')
+ // Add a new course
+  cy.contains("Courses").click()
+  cy.ustadAddCourse('003_005')
+ //Add a teacher
+  cy.contains("button","Members").click()
+  cy.contains("span","Add a teacher").click()
+  cy.ustadAddNewPerson('Teacher','1','Female')
+ // Add account for teacher
+  cy.contains("Teacher 1").click()
+  cy.contains('View profile').click()
+  cy.ustadCreateUserAccount('teacher1','test1234')
+ // Add module block
+  cy.contains('button','Course').click()
+  cy.contains('button','Edit').click()
+  cy.ustadAddModuleBlock('module 1')
+ // Add Content from library
+  cy.contains('button','Edit').click()
+  cy.contains("Add block").click()
+  cy.contains("Content").click()
+  cy.get('.MuiChip-label.MuiChip-labelMedium').eq(2).click() //Library button
```

Comment:
These css names are not auto-generated, but this test could fail if we change button order etc. Can we change this to use the text? I understand there is an issue that it could try and click library, but we could use .contains within an id.

I have added an id for the box that contains all the chips "content_entry_filter_chip_box" - so if you search for the text "library" within that box, it should get the right one.

Reply:
Done, Thanks